### PR TITLE
Missing fixes in #46

### PR DESCRIPTION
**Pull request by dashed**
Closes https://github.com/paypal/react-autocompletely/issues/32

Missing fixes in https://github.com/paypal/react-autocompletely/pull/46

Reference: https://github.com/paypal/react-autocompletely/pull/46#discussion_r130191005

### CODE REVIEW

File: src/input.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 
 import {AUTOCOMPLETE_CONTEXT} from './constants'
-import {compose, moveCursorToTheEnd} from './utils'
```
Comment (kentcdodds) [inline src/input.js]: Could you also remove this method from `utils`? It's not used anywhere else :smile:

File: src/input.js [hunk 2]
```diff
@@ -65,10 +65,6 @@ class Input extends Component {
     this.updateInputValue(event.target.value)
   }
 
-  reset = () => {
```
Comment (kentcdodds) [inline src/input.js]: Could you make sure that we're not calling `.reset()` on the input anywhere else? I think we may be...

File: src/input.js (hunk 1 above)
Comment (dashed) [inline src/input.js]: Resolved.

File: src/input.js (hunk 2 above)
Comment (dashed) [inline src/input.js]: Resolved.

**Comment by dashed** [comment]
Sorry I jumped the gun on the PR. 😅

**Pull request merged**